sw/airborne/firmwares/fixedwing/main_ap.c

```diff
@@ -425,6 +425,10 @@ static inline void telecommand_task(void)
 
     /** Pitch is bounded between [-AUTO1_MAX_PITCH;AUTO1_MAX_PITCH] */
     h_ctl_pitch_setpoint = FLOAT_OF_PPRZ(fbw_state->channels[RADIO_PITCH], 0., AUTO1_MAX_PITCH);
+#if H_CTL_YAW_LOOP
+    /** Yaw is bounded between [-AUTO1_MAX_YAW_RATE;AUTO1_MAX_YAW_RATE] */
+    h_ctl_yaw_rate_setpoint = 10 * FLOAT_OF_PPRZ(fbw_state->channels[RADIO_YAW], 0., AUTO1_MAX_YAW_RATE);
```

Comment:
The `FLOAT_OF_PPRZ` macro is scaling the radio input the `[-AUTO1_MAX_YAW_RATE; AUTO1_MAX_YAW_RATE]`, and it is compared afterwards to the yaw rate, so it should be with the same scaling.
So if the AUTO1_MAX_YAW_RATE value is really the maximum yaw rate you want to apply (in radian), there is no reason to multiply it.